airbyte-webapp/src/packages/cloud/views/layout/MainView/MainView.tsx

```diff
@@ -36,25 +38,51 @@ const DataBlock = styled.div<{ hasBanner?: boolean }>`
 `;
 
 const MainView: React.FC = (props) => {
+  const { formatMessage } = useIntl();
   const workspace = useCurrentWorkspace();
   const cloudWorkspace = useGetCloudWorkspace(workspace.workspaceId);
-  const showBanner =
+  const showCreditsBanner =
     cloudWorkspace.creditStatus &&
     [
       CreditStatus.NEGATIVE_BEYOND_GRACE_PERIOD,
       CreditStatus.NEGATIVE_MAX_THRESHOLD,
       CreditStatus.NEGATIVE_WITHIN_GRACE_PERIOD,
-    ].includes(cloudWorkspace.creditStatus);
+    ].includes(cloudWorkspace.creditStatus) &&
+    !cloudWorkspace.trialExpiryTimestamp;
```

Comment:
In this case, if the `trialExpiryTimestamp` is null-ish, then it means that it's not expired?

Reply:
If the `trialExpiryTimestamp` is null-ish then the user is not in a trial (that is how the API was configured by Cloud team).